src/widget/widget.js

```diff
@@ -52,6 +54,10 @@ function initWidget () {
         return
     }
 
+
+    const config = widgetConfigFromScriptTag()
```

Comment:
@guanzo to add some handling for when the DOM is not loaded yet, would `widgetConfigFromScriptTag()` return an error? That way can just add a try catch.

Reply:
sgtm